config/package/resources.yaml.gotmpl

```
@@ -54,6 +54,7 @@ spec:
       labels:
         app: validation-webhook
     spec:
+      automountServiceAccountToken: "false"
```

It complains that this is not a boolean due to the quotes.

```suggestion
      automountServiceAccountToken: false
```